src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -3217,6 +3217,98 @@ class AsAliasExpressionSegment(BaseSegment):
         Ref("SingleIdentifierGrammar"),
     )
 
+class ConflictActionSegment(BaseSegment):
+
+    type = "conflict_action"
+
+    match_grammar = Sequence(
+        "DO",
+        OneOf(
+            "NOTHING",
+            Sequence(
+                "UPDATE",
+                "SET",
+                Delimited(
+                    OneOf(
+                        Sequence(
+                            Ref("ColumnReferenceSegment"),
+                            Ref("EqualsSegment"),
+                            OneOf(
+                                Ref("ExpressionSegment"),
+                                "DEFAULT"
+                            )
+                        ),
+                        Sequence(
+                            Bracketed(
+                                Delimited(
+                                    Ref("ColumnReferenceExpression")
+                                )
+                            ),
+                            Ref("EqualsSegment"),
+                            Ref.keyword("ROW", optional=True),
+                            Bracketed(
+                                Delimited(
+                                    OneOf(
+                                        Ref("ExpressionSegment"),
+                                        "DEFAULT"
+                                    )
+                                )
+                            )
+                        ),
+                        # Sequence(
+                        #     Bracketed(
+                        #         Delimited(
+                        #             Ref("ColumnReferenceExpression")
+                        #         )
+                        #     ),
+                        #     Ref("EqualsSegment"),
+                        #     Bracketed(
+                        #         Ref("SubSelectExpression") # This isn't defined. Effectively an entire query.
```

Comment:
Would `Bracketed(Ref("SelectableGrammar")),` work for you here? There's a few other instances where we do that.

Reply:
Thank you!